src/routes/e2e/widget/modal/+page.svelte

```diff
@@ -3,6 +3,7 @@
   import { page } from '$app/stores';
 
   import Widget, { configuration, modal, journey, user } from '$package/modal';
+  import type { Response } from '$lib/widget/interfaces';
```

Comment:
Another file with svelte-check "fixes" in it that I needed to add in order to commit.

Reply:
Like the other comment. I consider the need for this a bug. The fix is implemented, but not sure which PR will include it.